test/e2e/user_management_test.go

```diff
@@ -45,7 +45,12 @@ func (s *userManagementTestSuite) TearDownTest() {
 }
 
 func (s *userManagementTestSuite) TestUserDeactivation() {
-	s.hostAwait.UpdateHostOperatorConfig(test.AutomaticApproval().Enabled())
+	s.hostAwait.UpdateHostOperatorConfig(
+		test.AutomaticApproval().Enabled(),
+		test.Deactivation().DeactivatingNotificationDays(-1))
```

Comment:
Sorry for my dumb question - why the negative number? wouldn't that mean that the deactivating notification will be send 1 day after deactivation?

Reply:
Ok, now I probably understand why you set the negative number - to skip the "wait" phase between deactivating & deactivation. I believe that better would be to modify the status part in the UserSignup "manually" as we do with `ProvisionedTime` in MUR.

Reply:
I'm not sure we can update `LastTransitionTime` the same way we can update `ProvisionedTime`, without having to modify the way that the `addOrUpdateStatusCondition()` function works in toolchain-common.

Reply:
You can update anything you want - you just need to call `Client.Status().Update()` to update the sub-resource. That would of course mean that you couldn't use the `addOrUpdateStatusCondition()` function - you would need to modify it manually.